### ISSUE TITLE
OXY4.0 : repeater issue that affects sliders

### ISSUE THREAD

**Issue by mikeingz** [soflyy/oxygen-bugs-and-features#2909]
In some case, the repeater element doesn't work well due of some change with the class **.oxy-dynamic-list**

Here is how the class looks like with **OXY3:**

```
.oxy-dynamic-list {
    width: 100%;
}
```

with **OXY4:**

```
.oxy-dynamic-list {
    width: 100%;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    flex-wrap: wrap;
}
```

Because of this change, the dynamic slider doesn't work anymore (and some other own made element)

Here is a temporary fix while waiting for a better solution : 

```
.oxy-dynamic-list {
    all: unset;
    height: inherit;
    width: 100%;
}
```

**Comment by mikeingz** [comment]
@Spellhammer 
After doing some tests, I can confirm that the Dynamic Slider is actually working fine. Sorry for that, it was some conflict with other sliders.
That being said, the fact that this class was modified (was it necessary to add all these flex property?) has bought some issues with own made elements.

**Comment by Spellhammer** [comment]
The changes to .oxy-dynamic-slider styles were not intended and were a side effect of another change we made. This has been fixed in Oxygen 4.0.1 RC 1, which is now available in the customer portal.
